src/bun.js/web_worker.zig

```diff
@@ -375,7 +375,11 @@ pub const WebWorker = struct {
             bun.outOfMemory();
         };
         JSC.markBinding(@src());
-        WebWorker__dispatchError(globalObject, worker.cpp_worker, bun.String.createUTF8(array.slice()), error_instance);
+        const handled = WebWorker__dispatchError(globalObject, worker.cpp_worker, bun.String.createUTF8(array.slice()), error_instance);
+        if (handled) {
+            vm.unhandled_error_counter -= 1;
```

Comment:
I'm not sure if this should be decremented

Reply:
It seemed suspect to me as well. I'll look at making a new handler instead.

Reply:
I'm not actually certain how else to do this without incrementing/decrementing whether it was handled or not. If you have thoughts, please lmk.